lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -591,11 +591,17 @@ public function getAlterTableSQL(TableDiff $diff)
                 }
             }
 
-            if ($columnDiff->hasChanged('comment')) {
+            $oldComment = null;
```

Comment:
Can the entire comment block be moved to a private method with correct nullable hints?

Reply:
do you mean extract the whole comment adding section to a method? i.e 594-606?

Reply:
ping @Ocramius

Reply:
@bendavies a `private function` that retrieves `$oldComment`

Reply:
done